sql/moz-fx-data-shared-prod/firefox_accounts_derived/fxa_oauth_events_v1/query.sql

```diff
@@ -28,7 +28,7 @@ WITH base AS (
       ) AS jsonPayload
     )
   FROM
-    `moz-fx-fxa-prod-0712.fxa_prod_logs.docker_fxa_oauth_20*`
+    `moz-fx-fxa-prod-0712.fxa_prod_logs.docker_fxa_oauth_20*`  -- # TODO: this table does not seem to exist anymore, should this query be deprecated?
```

Comment:
@jbuck can you confirm that fxa oauth events are gone, and we shouldn't expect them somewhere else or something like that?

Reply:
I can confirm that part, the source table is no longer present in BigQuery at all.